sonar-db/src/main/java/org/sonar/db/qualityprofile/QualityProfileDao.java

```diff
@@ -173,18 +172,12 @@ public void delete(int id) {
   }
 
   @CheckForNull
-  public List<QualityProfileDto> selectDefaultProfiles(final DbSession session, Collection<String> languageKeys) {
-    return DatabaseUtils.executeLargeInputs(languageKeys, new Function<List<String>, List<QualityProfileDto>>() {
-      @Override
-      @Nonnull
-      public List<QualityProfileDto> apply(@Nonnull List<String> input) {
-        return mapper(session).selectDefaultProfiles(input);
-      }
-    });
+  public List<QualityProfileDto> selectDefaultProfiles(DbSession session, Collection<String> languageKeys) {
+    return executeLargeInputs(languageKeys, input -> mapper(session).selectDefaultProfiles(input));
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Replace this lambda with a method reference. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1612)